extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -536,6 +536,10 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   getPreviousPageId_() {
+    if (this.element.hasAttribute('return-to')) {
```

Comment:
This is technically exposing a new API that savvy publishers could then start (ab)using.  I have a feeling you can get away without this, but if not, we should at least rename the attribute to `i-amphtml-return-to`, which makes it invalid for publishers to use this markup.  If we decide we actually want to open up this feature to publishers, we can just rename the property back, but that should be an explicit decision.

Reply:
if we move this to something like an `insertPage` method on amp story I think the need for this goes away

Reply:
so I refactored to `insertPage` but I am still using the (now prefixed) attributes. I am happy to change this method to manipulate the actual pages array in the next PR but I want to clarify a few implementation details with you first.